tests/linalg_test.py

```diff
@@ -1480,6 +1480,19 @@ def testRsf2csf(self, shape, dtype):
                               args_maker, tol=tol)
       self._CompileAndCheck(jsp.linalg.rsf2csf, args_maker)
 
+  @parameterized.named_parameters(
+        jtu.cases_from_list({
+            "testcase_name":
+            "_shape={}".format(jtu.format_shape_dtype_string(shape, dtype)),
+            "shape": shape, "dtype": dtype
+        } for shape in [(1, 1), (5, 5), (20, 20), (50, 50)]
+            for dtype in float_types + complex_types))
+  @jtu.skip_on_devices("gpu", "tpu")
+  def testFunm(self, shape, dtype):
+      rng = jtu.rand_default(self.rng())
+      args_maker = lambda: [rng(shape, dtype), lambda x: x**-2.718]
+      self._CheckAgainstNumpy(osp.linalg.funm, jsp.linalg.funm, args_maker)
```

Comment:
We should also call `_CompileAndCheck` once the above issues are addressed.

Reply:
I think I've now added this check in correctly (I followed the tests for `jnp.apply_along_axis` for inspiration).